src/main/java/org/gridsuite/study/server/SupervisionController.java

```diff
@@ -0,0 +1,70 @@
+package org.gridsuite.study.server;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.gridsuite.study.server.service.SupervisionService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/" + StudyApi.API_VERSION + "/supervision")
+@Tag(name = "Study server - Supervision")
+public class SupervisionController {
+    private final SupervisionService supervisionService;
+
+    public SupervisionController(SupervisionService supervisionService) {
+        this.supervisionService = supervisionService;
+    }
+
+    @DeleteMapping(value = "/loadflow/results")
+    @Operation(summary = "delete all studies loadflow results")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "all loadflow results have been deleted")})
+    public ResponseEntity<Void> deleteLoadflowResults() {
+        supervisionService.deleteLoadflowResults();
```

Comment:
It could be interesting to return number of deleted results ?

Reply:
Done